requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy==1.22.2
 protobuf==3.19.4
 tensorboard==2.8.0
-torch==1.12.1
\ No newline at end of file
+torch==1.13.0
```

Comment:
I tried to run both the training and testing scripts with this PyTorch version. They work fine.